documents/token.md

```diff
@@ -196,6 +196,62 @@ Customer customer = instance.customers.deleteToken(customerId, tokenId);
 }
 ```
 -------------------------------------------------------------------------------------------------------
+### Fetch VPA tokens of a customer id
+
+```java
+String customerId = "cust_DtHaBuooGHTuyZ";
+
+List<Customer> customer = instance.customers.fetchTokens(customerId);
```

Comment:
java: incompatible types: java.util.List<com.razorpay.Token> cannot be converted to java.util.List<com.razorpay.Customer>

Reply:
[fixed](https://github.com/razorpay/razorpay-java/blob/third_part_validation_token/documents/token.md?plain=1#L204)